### PR TITLE
quick-lint-js: Add version 2.4.1

### PR DESCRIPTION
**Pull request by strager**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/quick-lint-js.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2.3.1",
+    "description": "Find bugs in JavaScript programs.",
+    "homepage": "https://quick-lint-js.com/",
+    "license": "GPL-3.0-or-later,...",
```
Comment (rashil2000) [inline bucket/quick-lint-js.json:5]:
```suggestion
    "license": "GPL-3.0-or-later",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Excellent manifest!!!

File: bucket/quick-lint-js.json (hunk 1 above)
Comment (strager) [reply bucket/quick-lint-js.json:5]: I think it's incorrect to say that the quick-lint-js .exe is GPL-3.0-or-later. `quick-lint-js.exe` statically links Microsoft's CRT and some open source third-party libraries (such as Boost and simdjson). Details can be found in `share/doc/quick-lint-js/copyright` in the .zip-s.

Should I instead list all the licenses separately? I think the complete list (for the 2.3.1 release) is: GPLv3, MIT, BSL, Apache2.0, and Microsoft's CRT license.

Comment (rashil2000) [reply bucket/quick-lint-js.json:5]: > Should I instead list all the licenses separately?

Yes, separated by a `|` or `,`

Comment (rashil2000) [reply bucket/quick-lint-js.json:5]: Hey @strager! Any updates on this?

Comment (strager) [reply bucket/quick-lint-js.json:5]: I'm working on a release of quick-lint-js with slightly simpler licensing. I'll update this PR when the new release is ready.

File: bucket/quick-lint-js.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "64bit": {
+            "url": "https://c.quick-lint-js.com/releases/2.4.0/manual/windows.zip",
+            "hash": "sha256:4b7895e605b78f3d691e1ec5193c297aa71fbe8214b870e669c1263ad2030698"
+        },
+        "32bit": {
+            "url": "https://c.quick-lint-js.com/releases/2.4.0/manual/windows-x86.zip",
+            "hash": "sha256:bd43ec9fde0497b57fb7fa7d6b2ae98cb6b521689ef2372c29c8f75e21cb0c9c"
+        }
+    },
+    "bin": "bin/quick-lint-js.exe",
```
Comment (rashil2000) [inline bucket/quick-lint-js.json:16]:
```suggestion
    "bin": "bin\\quick-lint-js.exe",
```

**Pull request merged**